soc/riscv/riscv-ite/common/soc_espi.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2021 ITE Corporation. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _ITE_IT8XXX2_SOC_ESPI_H_
+#define _ITE_IT8XXX2_SOC_ESPI_H_
+
+#include <device.h>
+#include <sys/util.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define ESPI_IT8XXX2_SOC_DEV DEVICE_DT_GET(DT_NODELABEL(espi0))
+
+/**
+ * @brief eSPI input pad gating
+ *
+ * @param dev pointer to eSPI device
+ * @param enable/disable eSPI pad
+ */
+void espi_it8xxx2_pad_ctrl(const struct device *dev, bool enable);
```

Comment:
I suggest naming `void espi_it8xxx2_enable_pad_ctrl(const struct device *dev, bool enable)`.

This makes the caller invocation clearer:

```
espi_it8xxx_enable_pad_ctrl(dev, false);  // disables pads
espi_it8xxx_enable_pad_ctrl(dev, true);  // enables pads
```

Reply:
Done.